src/java/org/apache/cassandra/security/ThreadAwareSecurityManager.java

```diff
@@ -216,6 +214,7 @@ public void checkPermission(Permission perm)
 
         // required by JavaDriver 2.2.0-rc3 and 3.0.0-a2 or newer
         // code in com.datastax.driver.core.CodecUtils uses Guava stuff, which in turns requires this permission
+        // Evaluate removing this once the driver is removed as a dependency (see CASSANDRA-20326).
```

Comment:
@tolbertam prefix it with `TODO` maybe? Ides (idea) will mark it with a different color when detected as TODO so it catches an eye.

Reply:
good call, will do